test/msw-api/setup-server/scenarios/browser.mocks.ts

```diff
@@ -0,0 +1,7 @@
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
+const worker = setupServer(
```

Comment:
Could you please add a comment on top of this `setupServer` call that this is intentionally wrong code?
We are trying to keep test scenarios as actual usage examples, so it would be useful to mention this is _not_ how people should be using it.